gradle.properties

```diff
@@ -0,0 +1,6 @@
+//signing.keyId=<publicKey>
```

Comment:
OK, if it's just a template - would it be possible to publish without putting real credentials in this exactly file?